### ISSUE TITLE
Inner classes are not included

### ISSUE THREAD

**Issue by SonnyBonds** [lawmurray/doxide#17]
A struct or class nested inside another struct/class is not included in the output documentation, e.g:

```
/** A class */
class OuterClass
{
  /** An inner class - not recognized by Doxide */
  class InnerClass
  {
  };
};
```

**Comment by lawmurray** [comment]
Also now fixed, demo [here](https://doxide.org/demo/parsing/ClassWithMembers/). Thanks again for the report @SonnyBonds.

**Comment by lawmurray** [comment]
Included in new v0.2.1 release, [build underway](https://app.circleci.com/pipelines/github/lawmurray/download.indii.org/57/workflows/8c803766-8f7a-4679-a0ba-7a68f6c6300c).
